docs/02.API-REFERENCE.md

```diff
@@ -1824,6 +1824,11 @@ enough for the whole string.
 *Note*: Does not put '\0' to the end of string, the return value identifies
 the number of valid bytes in the output buffer.
 
+*Note*: If jerry value is larger than target buffer, then it will fail to copy.
```

Comment:
I would say: `If the size of the string in jerry value is larger than the size of the target buffer, the copy will fail.`